### PR TITLE
Text utilities & lists

### PR DESCRIPTION
**Pull request by msdeshmukh009**
**Features of Text Utilities** 
- Headings
- Text alignment
- Different text sizes
- Text transform
- Font weights
- Some extra styles.

**Features of lists**
- Unordered list styles
- Ordered list styles
- Inline list
- Stacked list

### CODE REVIEW

File: components/textUtilities/textUtilities.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  font-weight: 700;
+}
+.text-black {
+  font-weight: 900;
+}
+/*******/
+/*** extra-styles ***/
+.text-gray {
+  color: gray;
+}
```
Comment (Chakravarthi0) [inline components/textUtilities/textUtilities.css]: Just a suggestion:

text-gray, changes the font color. The user might think that text-black will change the font color to black. Rename  this class, if possible.

File: docsite-css/component-display-conatiner.css [hunk 2]
```diff
@@ -31,3 +31,21 @@
...
+.text-utility-container {
+  display: flex;
+  row-gap: 1rem;
+  flex-direction: column;
+  margin: 0 auto;
+}
+.stcked-list-conatainer {
+  margin: 0 auto;
+  border: 2px solid var(--secondary-color);
+  border-radius: 0.3rem;
```
Comment (Chakravarthi0) [inline docsite-css/component-display-conatiner.css:49]: Vikram has suggested to used px units when the value we want to use is less than 10px. here you could you 3px for border-radius. 

 You could use either px or rem, just keep things consistent.

Rest looks good,

**Review by Chakravarthi0 — APPROVED** [review]

File: components/textUtilities/textUtilities.css (hunk 1 above)
Comment (msdeshmukh009) [reply components/textUtilities/textUtilities.css]: Ohh I did not realize that. Thanks, I will change it.

File: docsite-css/component-display-conatiner.css (hunk 2 above)
Comment (msdeshmukh009) [reply docsite-css/component-display-conatiner.css:49]: Yes, I think I have made same mistake in many places. I think I will refactor them together. Thanks for pointing it out.

**Review by srishti-maurya — APPROVED** [review]
list UI looks great

**Pull request merged**